pytorch_pfn_extras/dataset/tabular/_transform.py

```diff
@@ -2,7 +2,7 @@
 from pytorch_pfn_extras.dataset.tabular import _utils
 
 
-class _Transform(tabular_dataset.TabularDataset):
+class _TransformBase:
```

Comment:
How about making `_TransformBase` as inheritance of `TabularDataset`? It can avoid multiple inheritance in `_Transform` and `_TransformBatch`.

Reply:
Yes, this makes more sense, thanks!